examples/api-implementation.js

```diff
@@ -0,0 +1,15 @@
+const catFactButton = document.getElementById('button');
+const catFactsList = document.getElementById('unordered-list');
+catFactButton.onclick = function () { getCatFactOnClick(); };
+async function getCatFactOnClick() {
```

Comment:
**suggestion:** the compiled files (of any kind) are usually ignored in the .gitignore